@generated
Call dependencies
exception_test.$toplevel (fun) -> []
exception_test.none_throws (fun) -> [object::__init__ (method) object::__new__ (method)]
exception_test.test_before_try_to_finally (fun) -> [_test_sink (fun) _test_source (fun) exception_test.none_throws (fun)]
exception_test.test_caught_exception (fun) -> [BaseException::__str__ (method) isinstance (fun)]
exception_test.test_constructed_exception (fun) -> [BaseException::__str__ (method) object::__init__ (method) object::__new__ (method)]
exception_test.test_except_to_finally (fun) -> [_test_sink (fun) _test_source (fun) exception_test.none_throws (fun)]
exception_test.test_parameter_flow (fun) -> [BaseException::__str__ (method)]
exception_test.test_return_finally (fun) -> [_test_source (fun) print (fun)]
exception_test.test_return_overrides_finally (fun) -> [_test_source (fun)]
exception_test.test_return_twice_finally (fun) -> [_test_source (fun)]
exception_test.test_sink_in_finally (fun) -> [_test_sink (fun) exception_test.none_throws (fun)]
exception_test.test_within_try_to_finally (fun) -> [_test_sink (fun) _test_source (fun) exception_test.none_throws (fun)]
